### ISSUE TITLE
Problems on gluetun with SERVER_COUNTRIES

### ISSUE THREAD

**Issue by tom9123** [qdm12/gluetun#1631]
I have problems in Unraid with gluetun Docker with the SERVER_COUNTRIES: Switzerland or Germany. No country works anymore. The countries are available in the servers.json

```
Running version latest built on 2023-05-30T13:06:28.040Z (commit b87b210)

🔧 Need help? https://github.com/qdm12/gluetun/discussions/new
🐛 Bug? https://github.com/qdm12/gluetun/issues/new
✨ New feature? https://github.com/qdm12/gluetun/issues/new
☕ Discussion? https://github.com/qdm12/gluetun/discussions/new
💻 Email? quentin.mcgaw@gmail.com
💰 Help me? https://www.paypal.me/qmcgaw https://github.com/sponsors/qdm12
2023-05-30T18:39:21Z INFO [routing] default route found: interface eth0, gateway 172.17.0.1, assigned IP 172.17.0.2 and family v4
2023-05-30T18:39:21Z INFO [routing] local ethernet link found: eth0
2023-05-30T18:39:21Z INFO [routing] local ipnet found: 172.17.0.0/16
2023-05-30T18:39:21Z INFO [firewall] enabling...
2023-05-30T18:39:21Z INFO [firewall] enabled successfully
2023-05-30T18:39:22Z INFO [storage] merging by most recent 13056 hardcoded servers and 13056 servers read from /gluetun/servers.json
2023-05-30T18:39:22Z ERROR VPN settings: provider settings: server selection: for VPN service provider nordvpn: the country specified is not valid: one or more values is set but there is no possible value available
2023-05-30T18:39:22Z INFO Shutdown successful
========================================
========================================
=============== gluetun ================
========================================
=========== Made with ❤️ by ============
======= https://github.com/qdm12 =======
========================================
========================================
```

**Comment by STRAYKR** [comment]
Same, but with PureVPN. no matter what country I try it says the country is not valid. It seems to be an issue with case maybe... it says 'switzerland' or 'netherlands' is not valid, doesn't matter which I set it to, but a valid entry is 'Switzerland' or "Netherlands' from the list it provides. I have it set with a capital letter in my docker compose config file, and it was working up until very recently.

**Comment by lemstress** [comment]
Same exact issue here. Ended up rolling back to v3.33 and it went back to normal.

**Comment by qdm12** [comment]
@tom9123 you need to use `SERVER_REGIONS`, not `SERVER_COUNTRIES`, as described in https://github.com/qdm12/gluetun/wiki/NordVPN. Please also share the logs next time as requested in the issue template, that could had saved me a good 10 minutes digging in the code 😉 

@STRAYKR @lemstress @hlugt latest image had issues 2 days ago (due to 47593928f9afe6456c53490fd1f1d6c9331730b4), which got fixed yesterday (with 2c30984a10bd0667a715fbd5f77b268a58a0af42) make sure you `docker pull qmcgaw/gluetun` 😉 You may be running the bad commit build 😢 

@lemstress you might want to use `v3.34` (or `v3` and pull it) instead of `v3.33` 😉

**Comment by tom9123** [comment]
Again after update problem with the same settings:

<img width="1065" alt="grafik" src="https://github.com/qdm12/gluetun/assets/135061009/d957a674-cb0b-4609-b641-8e95e1f062ec">
